pkgs/os-specific/linux/nvidia-x11/builder.sh

```diff
@@ -3,7 +3,7 @@ source $stdenv/setup
 
 unpackManually() {
     skip=$(sed 's/^skip=//; t; d' $src)
-    tail -n +$skip $src | xz -d | tar xvf -
+    tail -n +$skip $src | bsdtar xvf -
```

Comment:
This one unfortunatelly broke my installation of nixGL with NVIDIA-Linux-x86_64-470.239.06 driver
```
Terminated
/nix/store/pxvl01q3az1543jx1vzzj3ap2jkpkhx5-builder.sh: line 6: bsdtar: command not found
tail: error writing 'standard output': Broken pipe
```

Reply:
installation error
```
jankrejci@station:~/.config/home-manager$ home-manager switch --impure 
error: builder for '/nix/store/skwg6inwprh9wmkvx5vi2rb9y014p8y3-nvidia-x11-470.239.06-nixGL.drv' failed with exit code 127;
       last 11 log lines:
       > Running phase: unpackPhase
       > tail: error writing 'standard output': Broken pipe
       > Creating directory NVIDIA-Linux-x86_64-470.239.06
       > Verifying archive integrity... OK
       > Uncompressing NVIDIA Accelerated Graphics Driver for Linux-x86_64 470.239.06/nix/store/vmm1c1bwzn9s5m8dgcwlviym7n1lkhwm-NVIDIA-Linux-x86_64-470.239.06.run: line 728: /build/makeself.XjK0VJG0/xz: cannot execute: required file not found
       > tail: error writing 'standard output': Broken pipe
       > /nix/store/vmm1c1bwzn9s5m8dgcwlviym7n1lkhwm-NVIDIA-Linux-x86_64-470.239.06.run: line 719: /dev/tty: No such device or address
       >
       > Terminated
       > /nix/store/pxvl01q3az1543jx1vzzj3ap2jkpkhx5-builder.sh: line 6: bsdtar: command not found
       > tail: error writing 'standard output': Broken pipe
       For full logs, run 'nix log /nix/store/skwg6inwprh9wmkvx5vi2rb9y014p8y3-nvidia-x11-470.239.06-nixGL.drv'.
error: 1 dependencies of derivation '/nix/store/rhwjvy67xap07260yh5p0p6qrwbpa3yg-nixGLNvidia-470.239.06.drv' failed to build
error: 1 dependencies of derivation '/nix/store/vgg6m9py1ydn8ddypxbjjbd95ac6fp9p-nixGL.drv' failed to build
error: 1 dependencies of derivation '/nix/store/0ngc4rbxsdkm7r9mkm8sdqz18xvi3qs6-home-manager-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/pyc8vmfvdrjhrvjs07sbwd42sly9gbri-home-manager-generation.drv' failed to build
```

log file
```
jankrejci@station:~/.config/home-manager$ nix log /nix/store/skwg6inwprh9wmkvx5vi2rb9y014p8y3-nvidia-x11-470.239.06-nixGL.drv
warning: The interpretation of store paths arguments ending in `.drv` recently changed. If this command is now failing try again with '/nix/store/skwg6inwprh9wmkvx5vi2rb9y014p8y3-nvidia-x11-470.239.06-nixGL.drv^*'
@nix { "action": "setPhase", "phase": "unpackPhase" }
Running phase: unpackPhase
tail: error writing 'standard output': Broken pipe
Creating directory NVIDIA-Linux-x86_64-470.239.06
Verifying archive integrity... OK
Uncompressing NVIDIA Accelerated Graphics Driver for Linux-x86_64 470.239.06/nix/store/vmm1c1bwzn9s5m8dgcwlviym7n1lkhwm-NVIDI>
tail: error writing 'standard output': Broken pipe
/nix/store/vmm1c1bwzn9s5m8dgcwlviym7n1lkhwm-NVIDIA-Linux-x86_64-470.239.06.run: line 719: /dev/tty: No such device or address

Terminated
/nix/store/pxvl01q3az1543jx1vzzj3ap2jkpkhx5-builder.sh: line 6: bsdtar: command not found
tail: error writing 'standard output': Broken pipe
```